presto/presto.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+#    Copyright 2015 Google, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+set -x -e
+
+# Variables for running this script
+ROLE=$(/usr/share/google/get_metadata_value attributes/role)
+HOSTNAME=$(hostname)
+DNSNAME=$(dnsdomainname)
+FQDN=${HOSTNAME}.$DNSNAME
+CONNECTOR_JAR=$(find /usr/lib/hadoop/lib -name 'gcs-connector-*.jar')
+PRESTO_VERSION="0.126"
+HTTP_PORT="8080"
+
+# Download and unpack Presto server
+wget https://repo1.maven.org/maven2/com/facebook/presto/presto-server/${PRESTO_VERSION}/presto-server-${PRESTO_VERSION}.tar.gz
+tar -zxvf presto-server-${PRESTO_VERSION}.tar.gz
+mkdir /var/presto
+mkdir /var/presto/data
+
+# Copy required Jars
+cp ${CONNECTOR_JAR} presto-server-${PRESTO_VERSION}/plugin/hive-hadoop2
+
+# Configure Presto
+mkdir presto-server-${PRESTO_VERSION}/etc
+mkdir presto-server-${PRESTO_VERSION}/etc/catalog
+
+cat > presto-server-${PRESTO_VERSION}/etc/node.properties <<EOF
+node.environment=production
+node.id=$(uuidgen)
+node.data-dir=/var/presto/data
+EOF
+
+cat > presto-server-${PRESTO_VERSION}/etc/catalog/hive.properties <<EOF
+connector.name=hive-hadoop2
+hive.metastore.uri=thrift://localhost:9083
```

Comment:
Probably fine to start with this, but add a TODO to keep track of switching this over to pulling it from /etc/hive/conf/hite-site.xml